main.js

```diff
@@ -1 +1,25 @@
+var level = require("level");
 var db  = level("./student-db", { valueEncoding: 'json'});
+
+
+const prompt = require('prompt-sync')({sigint: true});
+
+ var stId = prompt('Enter ID: ');
+ var fullName = prompt('Enter full Name: ');
+ var stdAge = prompt('enter age:');
+ var stdAddress = prompt('Enter address: ');
+
+acceptStudents(stId, fullName,stdAge, stdAddress);
+
+function acceptStudents(stId, fullName,stdAge){
+    db.put('student', {id: stId ,name: fullName, age: stdAge, address: stdAddress, status: 'applying' }, function(err){  
```

Comment:
this code would only allow 1 student to be stored in the database 
you use the stId as key like
```js
db.put(stId, { id: stId, name: fullName, age: stdAge, address: stdAddress, status: 'applying' }
```
Bug: your variable stdAddress is not defined in the arguments